### DISCUSSION TITLE
Pedestal scalings using published data

### DISCUSSION THREAD

**Discussion by jonmaddock** [ukaea/PROCESS#2931] [Ideas]
The DEMO pedestal scaling has been removed from the open source version of PROCESS as it was based on EUROfusion data.  

The best we can get to a published version is [here](https://www.sciencedirect.com/science/article/pii/S0920379622000473?via%3Dihub#fig0004).

If we want a scaling based on real published data we could try these papers:  
[Role of the separatrix density in the pedestal performance in deuterium low triangularity JET-ILW plasmas and comparison with JET-C](https://iopscience.iop.org/article/10.1088/1741-4326/ac3363/meta) (26 pages) , and  

[Pedestal structure, stability and scalings in JET-ILW: the EUROfusion JET-ILW pedestal database](https://iopscience.iop.org/article/10.1088/1741-4326/abb79e) (50 pages),  
both by Lorenzo Frassinetti.

-------------------------
Remember also that using such scalings can give perverse results.  If, for example, you increase the pedestal density, but you don't change the volume-average density, the code will reduce the central density, which is unphysical.  (The central density can even become negative - see #736.)
![image](https://github.com/ukaea/PROCESS/assets/9718416/b7f5150f-32dd-406b-92dc-64eeb9f7ca71)

Note that the behaviour is different with the central temperature, which is given by a completely different formula:
![image](https://github.com/ukaea/PROCESS/assets/9718416/6b4747d1-3c89-4c09-ae02-96ea5ab846e8)
$`T_{ped}`$ appears with a plus sign at the beginning of the formula, but the sign of the second term depends on the sign of $`\gamma`$.

**Comment by chris-ashe** [comment]
Note that the central temperature function has been found to be incorrect as is now replaced by
![image](https://github.com/user-attachments/assets/bca1fbf9-abf1-4ed1-8585-4973646cb5f0)
